components/bl-innerImageZoom-component/README.md

```diff
@@ -0,0 +1,51 @@
+# InnerImageZoom
+
+InnerImageZoom is the component that can be used in Backendless [UI-Builder](https://backendless.com/developers/#ui-builder). It allows the user to apply an image zoom effect inside that image in the application.
+
+<p align="center">
+  <img src="./thumbnail.png" alt="main thumbnail" width="780"/>
+</p>
+
+## Configuration
+
+Configuration can be done in the UI Builder or using the Codeless Logic. You have to specify the source of the image that will be displayed in the component and to which the zoom effect will be applied.
+
+## Properties
+
+| Property                   | Type     | Default value                                                                                                                        | Logic              | Data Binding | UI Setting | Description                           |
+|----------------------------|----------|--------------------------------------------------------------------------------------------------------------------------------------|--------------------|--------------|------------|---------------------------------------|
+| Image Source <br> `source` | *Text*   | `https://content.r9cdn.net/rimg/dimg/6d/56/5bd0abd4-city-26896-1670df92320.jpg?width=1366&height=768&xhint=1567&yhint=925&crop=true` | Image Source Logic | YES          | YES        | Specifies the image source to zoom.   |
+| Zoom <br> `zoomIn`         | *Number* | 2                                                                                                                                    |                    | NO           | YES        | Controlled zoom to enlarge the image. |
```

Comment:
Would be better to use percents for that. 140 looks for me better than 1.4(or 1,4)